index.html

```diff
@@ -617,6 +617,20 @@
           </p>
         </div>
         <p>
+          The following algorithm provides an <dfn>extension point</dfn>: other
+          specifications that define more specific algorithms for matching the available 
+          <a>payment apps</a> to the provided <a>payment methods</a> are encouraged to
+          hook themselves into this specification at this point in the
+          algorithm.
+        </p>
+        <div class="note">
```

Comment:
I suggest we remove this. I know I have it in the manifest spec, but I added this there originally because we were trying to understand the implications and process of monkey patching things back in 2014 - specially across disparate specs and standards orgs. Anne's post was certainly helpful in providing guidance. 

In our case, we are basically adding an extension point for specs that we will work on just within this working group.